package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "braze-segment-debounce",
```

Comment:
Started a convo in our DMs w/ Eric about whether or not we want to publish this scoped to the vivianhealth org

Reply:
Per our convo:

```suggestion
  "name": "@vivianhealth/braze-segment-debounce",
```